### DISCUSSION TITLE
LESSON 10: depositing function time out.

### DISCUSSION THREAD

**Discussion by elmbark2023** [smartcontractkit/full-blockchain-solidity-course-py#449] [Q&A] [answered]
Hi All, 
I have encountered with an error when trying the aave_brownie.py scripts. actually I am using the codes from git, but it still fails. I have attached the screenshot and hopefully someone could shed some light.  Thanks a lot!
![image](https://user-images.githubusercontent.com/1181400/143725532-71d5ba4d-ad68-493a-8d83-f7cab2ba04c2.png)
![image](https://user-images.githubusercontent.com/1181400/143725546-be465e5d-fa26-4ec9-a7d5-72276fcec105.png)

**Comment by elmbark2023** [comment, marked as answer]
I fixed it by increasing the timeout for the mainnet-fork network to 120 from the default 30 seconds. 

However I am not sure why I was having this issue, maybe my local machine(MID2014 MBP) is too slow...

**Reply by cromewar** [reply]
Hi @ziyuan according to [this](https://kite.trade/forum/discussion/7712/max-retries-failing-seems-like-timeout-while-placing-order#latest) thread: 

Timeout exceptions are very common while dealing with HTTP calls. If it is a read request then you can retry but if it is a write request like order place, it is better to check orderbook before doing the retry otherwise you might end up placing multiple orders.

I think setting the time higher cause the error on your code.

Best regards, Cromewar.
